akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -132,20 +131,31 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
     expectMessage(max.asScala, hint, obj)
 
   private def expectMessage_internal[T <: M](max: Duration, obj: T, hint: Option[String] = None): T = {
-    val o = receiveOne(max)
+    val o = receiveOne_internal(max)
     val hintOrEmptyString = hint.map(": " + _).getOrElse("")
     assert(o != null, s"timeout ($max) during expectMessage while waiting for $obj" + hintOrEmptyString)
     assert(obj == o, s"expected $obj, found $o" + hintOrEmptyString)
     o.asInstanceOf[T]
   }
 
+  override def receiveOne(): M = receiveOne(remainingOrDefault)
+
+  override def receiveOne(max: java.time.Duration): M = receiveOne(max.asScala)
+
+  def receiveOne(max: Duration): M = {
+    val o = receiveOne_internal(remainingOrDefault)
```

Comment:
Someone in the call chain needs to dilate the timeout, maybe here?

Reply:
@patriknw ^^? Originally I had a dilated but with the addition of the second receiveOne(max: FiniteDuration) there was no longer an exposed non-dilated and a dilated to choose from. Is dilated the base expected for all 'receiveOne's?